### ISSUE TITLE
[Bug]: Everybody gets marked as mods users by protection? 

### ISSUE THREAD

**Issue by MrNoBrain** [Yimura/YimMenu#700]
### Describe the bug

Hello the protection system might me bugged it protects too much it marks every body as cheatrs  sending one screenshot of the notification

### Steps To Reproduce

1. trun on any protection
2. everybody get marked as other mods users

### Expected Behavior

not to get everybody marked as modders
even if they don't have mod like my friends 

### Screenshots and/or Logs

```
<paste your logs here>
```
![image](https://user-images.githubusercontent.com/120467953/207669542-a175760c-a367-4839-8772-8faf27a639ed.png)

### Store

Rockstar

### Additional context

Still doing the same (16.12.)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by MrNoBrain** [comment]
Thats what it says to about player

**Comment by Yimura** [comment]
Could you post your settings.json here? When I tested YimMenu this did not happen.

**Comment by MrNoBrain** [comment]
> Could you post your settings.json here? When I tested YimMenu this did not happen.

I would love to, but idk how i got only dll and im kinda new to github

**Comment by Yimura** [comment]
> > Could you post your settings.json here? When I tested YimMenu this did not happen.
> 
> I would love to idk how i got only dll
> 

You can find it here `%appdata%/BigBaseV2/settings.json`

**Comment by MrNoBrain** [comment]
> > > Could you post your settings.json here? When I tested YimMenu this did not happen.
> > 
> > 
> > I would love to idk how i got only dll
> 
> You can find it here `%appdata%/BigBaseV2/settings.json`

sry dont have BigBaseV2 and cannot open it if I try to by cmd

**Comment by SkiddyToast** [comment]
Please press the Windows key + R to open the "Run" box, then type in `%appdata%`. Then navigate to the folder labeled 'BigBaseV2', and open the `settings.json` file with your favorite text editor, such as notepad or Visual Studio Code.

**Comment by MrNoBrain** [comment]
```json
{
    "clone_pv": {
        "clone_plate": false,
        "plate": "",
        "preview_vehicle": true,
        "spawn_clone": false,
        "spawn_inside": true,
        "spawn_maxed": false
    },
    "context_menu": {
        "allowed_entity_types": 15,
        "bounding_box_color": 4278255360,
        "bounding_box_enabled": true,
        "enabled": false,
        "selected_option_color": 4278255360
    },
    "debug": {
        "logs": {
            "metric_logs": false,
            "script_hook_logs": true
        }
    },
    "esp": {
        "armor": true,
        "box": true,
        "box_render_distance": [
            0.0,
            1500.0
        ],
        "change_esp_color_from_dist": false,
        "default_color": 4285713522,
        "distance": true,
        "distance_threshold": [
            0.0,
            0.0
        ],
        "enabled": false,
        "enemy_color": 4281479904,
        "enemy_near_color": 4283794943,
        "friend_color": 4293244509,
        "global_render_distance": [
            0.0,
            1500.0
        ],
        "god": true,
        "health": true,
        "hide_self": true,
        "name": true,
        "scale_armor_from_dist": false,
        "scale_health_from_dist": false,
        "tracer": false,
        "tracer_draw_position": [
            0.5,
            1.0
        ],
        "tracer_render_distance": [
            0.0,
            1500.0
        ]
    },
    "notifications": {
        "gta_thread_kill": {
            "log": false,
            "notify": false
        },
        "gta_thread_start": {
            "log": true,
            "notify": false
        },
        "invalid_sync": {
            "log": false,
            "notify": true
        },
        "mismatch_sync_type": {
            "log": false,
            "notify": false
        },
        "network_player_mgr_init": {
            "log": false,
            "notify": true
        },
        "network_player_mgr_shutdown": {
            "log": true,
            "notify": true
        },
        "out_of_allowed_range_sync_type": {
            "log": false,
            "notify": false
        },
        "player_join": {
            "above_map": true,
            "log": true,
            "notify": true
        },
        "player_leave": {
            "log": false,
            "notify": false
        },
        "received_event": {
            "clear_ped_task": {
                "log": false,
                "notify": false
            },
            "kick_vote": {
                "log": false,
                "notify": true
            },
            "modder_detect": {
                "log": true,
                "notify": true
            },
            "report_cash_spawn": {
                "log": false,
                "notify": false
            },
            "request_control_event": {
                "log": false,
                "notify": true
            },
            "vehicle_temp_action": {
                "log": false,
                "notify": false
            }
        },
        "reports": {
            "log": false,
            "notify": true
        },
        "script_event_handler": {
            "bounty": {
                "log": true,
                "notify": true
            },
            "ceo_ban": {
                "log": true,
                "notify": true
            },
            "ceo_kick": {
                "log": true,
                "notify": true
            },
            "ceo_money": {
                "log": false,
                "notify": true
            },
            "clear_wanted_level": {
                "log": false,
                "notify": true
            },
            "crash": {
                "log": false,
                "notify": true
            },
            "fake_deposit": {
                "log": false,
                "notify": true
            },
            "force_mission": {
                "log": false,
                "notify": true
            },
            "force_teleport": {
                "log": false,
                "notify": true
            },
            "gta_banner": {
                "log": false,
                "notify": true
            },
            "mc_teleport": {
                "log": false,
                "notify": true
            },
            "network_bail": {
                "log": false,
                "notify": true
            },
            "null_function_kick": {
                "log": false,
                "notify": true
            },
            "personal_vehicle_destroyed": {
                "log": false,
                "notify": true
            },
            "remote_off_radar": {
                "log": false,
                "notify": true
            },
            "rotate_cam": {
                "log": false,
                "notify": true
            },
            "send_to_cutscene": {
                "log": false,
                "notify": true
            },
            "send_to_location": {
                "log": false,
                "notify": true
            },
            "sound_spam": {
                "log": false,
                "notify": true
            },
            "spectate": {
                "log": false,
                "notify": true
            },
            "start_activity": {
                "log": false,
                "notify": true
            },
            "switch_player_model": {
                "log": false,
                "notify": false
            },
            "teleport_to_warehouse": {
                "log": false,
                "notify": true
            },
            "transaction_error": {
                "log": false,
                "notify": true
            },
            "tse_freeze": {
                "log": false,
                "notify": true
            },
            "tse_sender_mismatch": {
                "log": false,
                "notify": true
            },
            "vehicle_kick": {
                "log": false,
                "notify": true
            }
        },
        "send_net_info_to_lobby": {
            "log": false,
            "notify": false
        },
        "transaction_rate_limit": {
            "log": false,
            "notify": true
        }
    },
    "protections": {
        "lessen_breakups": true,
        "rid_join": true,
        "script_events": {
            "bounty": true,
            "ceo_ban": true,
            "ceo_kick": true,
            "ceo_money": true,
            "clear_wanted_level": true,
            "crash": true,
            "fake_deposit": true,
            "force_mission": true,
            "force_teleport": true,
            "gta_banner": true,
            "mc_teleport": true,
            "network_bail": true,
            "personal_vehicle_destroyed": true,
            "remote_off_radar": true,
            "rotate_cam": true,
            "send_to_cutscene": true,
            "send_to_location": true,
            "sound_spam": true,
            "spectate": true,
            "start_activity": true,
            "switch_player_model": true,
            "teleport_to_warehouse": true,
            "transaction_error": true,
            "vehicle_kick": true
        },
        "script_host_kick": true
    },
    "self": {
        "auto_tp": false,
        "clean_player": false,
        "fast_respawn": false,
        "force_show_hud": false,
        "force_show_hud_element": false,
        "god_mode": false,
        "hide_ammo": false,
        "hide_radar": false,
        "hud_components_states": [
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false
        ],
        "invisibility": false,
        "local_visibility": false,
        "mobile_radio": false,
        "never_wanted": false,
        "no_collision": false,
        "no_ragdoll": false,
        "no_water_collision": false,
        "off_radar": false,
        "proof_bullet": false,
        "proof_collision": false,
        "proof_drown": false,
        "proof_explosion": false,
        "proof_fire": false,
        "proof_mask": 0,
        "proof_melee": false,
        "proof_steam": false,
        "proof_water": false,
        "selected_hud_component": 1,
        "super_run": false,
        "unlimited_oxygen": false
    },
    "session": {
        "advertise_menu": false,
        "block_ceo_money": false,
        "damage_karma": false,
        "decloak_players": false,
        "disable_chat_filter": false,
        "disable_peds": false,
        "disable_traffic": false,
        "explosion_karma": false,
        "force_script_host": false,
        "force_session_host": true,
        "force_thunder": false,
        "is_team": false,
        "join_in_sctv_slots": false,
        "kick_chat_spammers": true,
        "kick_host_when_forcing_host": true,
        "log_chat_messages": false,
        "log_text_messages": false,
        "name_spoof_enabled": false,
        "player_magnet_count": 32,
        "player_magnet_enabled": false,
        "randomize_ceo_colors": false,
        "send_to_apartment_idx": 1,
        "send_to_warehouse_idx": 1,
        "spoofed_name": ""
    },
    "session_browser": {
        "language_filter": 0,
        "language_filter_enabled": false,
        "player_count_filter_enabled": false,
        "player_count_filter_maximum": 32,
        "player_count_filter_minimum": 0,
        "region_filter": 3,
        "region_filter_enabled": true,
        "replace_game_matchmaking": false,
        "sort_direction": 0,
        "sort_method": 0
    },
    "settings": {
        "dev_dlc": false,
        "hotkeys": {
            "menu_toggle": 45,
            "noclip": 0,
            "teleport_objective": 117,
            "teleport_waypoint": 116
        }
    },
    "spawn_ped": {
        "preview_ped": true
    },
    "spawn_vehicle": {
        "plate": "",
        "preview_vehicle": true,
        "spawn_inside": true,
        "spawn_maxed": true
    },
    "spoofing": {
        "crew_tag": "",
        "ip_address": [
            42,
            42,
            42,
            42
        ],
        "rockstar_crew": false,
        "rockstar_id": 0,
        "session_language": 0,
        "session_player_count": 25,
        "session_region_type": 0,
        "spoof_cheater": false,
        "spoof_crew_data": false,
        "spoof_hide_god": true,
        "spoof_hide_spectate": true,
        "spoof_ip": true,
        "spoof_rockstar_dev": false,
        "spoof_rockstar_id": false,
        "spoof_rockstar_qa": false,
        "spoof_session_language": false,
        "spoof_session_player_count": false,
        "spoof_session_region_type": false,
        "spoof_username": false,
        "square_crew_tag": false,
        "username": ""
    },
    "tunables": {
        "disable_phone": false,
        "no_idle_kick": false
    },
    "vehicle": {
        "auto_drive_speed": 149.98434448242188,
        "auto_drive_style": 1,
        "auto_turn_signals": false,
        "boost_behavior": 0,
        "change_engine_state_immediately": false,
        "disable_engine_auto_start": false,
        "drive_on_water": false,
        "fly": {
            "dont_stop": false,
            "enabled": false,
            "no_collision": false,
            "speed": 1.0,
            "stop_on_exit": false
        },
        "god_mode": false,
        "horn_boost": false,
        "instant_brake": false,
        "is_targetable": true,
        "keep_vehicle_repaired": false,
        "no_water_collision": false,
        "proof_bullet": false,
        "proof_collision": false,
        "proof_explosion": false,
        "proof_fire": false,
        "proof_mask": 0,
        "proof_melee": false,
        "proof_steam": false,
        "proof_water": false,
        "rainbow_paint": {
            "neon": true,
            "primary": false,
            "secondary": false,
            "smoke": false,
            "speed": 0,
            "type": 0
        },
        "seatbelt": false,
        "speed_unit": 1,
        "speedo_meter": {
            "enabled": false,
            "left_side": false,
            "position_x": 0.8999999761581421,
            "position_y": 0.7200000286102295
        },
        "turn_signals": false,
        "vehicle_jump": false
    },
    "weapons": {
        "ammo_special": {
            "toggle": false,
            "type": 0
        },
        "bypass_c4_limit": false,
        "custom_weapon": 0,
        "force_crosshairs": false,
        "increased_damage": 1.0,
        "infinite_ammo": false,
        "infinite_mag": false,
        "no_recoil": false,
        "no_spread": false,
        "rapid_fire": false
    },
    "window": {
        "color": 3357612055,
        "gui_scale": 1.0
    }
}
```

**Comment by MrNoBrain** [comment]
well... thats it, thanks for help with how to do it and sorry if u wanted me to post it as file

**Comment by Yimura** [comment]
No like that is perfect

**Comment by Adler-D3R** [comment]
Does it mark the entire session as modder or only some players ? Be aware that Sync Protections can send false positives sometimes. However, if you encounter this issue real quick in every sessions, that's not normal.

**Comment by maybegreat48** [comment]
The node class broke in the recent update causing the values to be corrupted and thus leading to false positives

**Comment by MrNoBrain** [comment]
> Does it mark the entire session as modder or only some players ? Be aware that Sync Protections can send false positives sometimes. However, if you encounter this issue real quick in every sessions, that's not normal.

only happening when there is activity like prep mission or supply mission
everybody that do prep or supply get marked that what i figured out

**Comment by Gh0sSte** [comment]
can someone please fix this? My entire screen is constantly flooded with errors..

**Comment by Yimura** [comment]
> can someone please fix this? My entire screen is constantly flooded with errors..

Is it this specific modder detection?

**Comment by Gh0sSte** [comment]
yes... I am sorry for my late response. It is saying that someone is trying to bounty me and inventory crash me. I am guessing some of the script events need updating as the old ones have been given another purpose?

**Comment by SkiddyToast** [comment]
The inventory crash notification is probably from the new weapons being added.

**Comment by SkiddyToast** [comment]
Never mind, the numbers haven't changed this update.

**Comment by Yimura** [comment]
Should've been fixed with #706
